libkbfs/test_common.go

```diff
@@ -79,9 +79,10 @@ func MakeTestBlockServerOrBust(t logger.TestLogBackend,
 	}
 }
 
-// MakeTestConfigOrBust creates and returns a config suitable for
-// unit-testing with the given list of users.
-func MakeTestConfigOrBust(t logger.TestLogBackend,
+// MakeTestConfigOrBustLoggedIn creates and returns a config suitable for
+// unit-testing with the given list of users. loggedInIndex specifies the
+// index (in the list) of the user being logged in
+func MakeTestConfigOrBustLoggedIn(t logger.TestLogBackend, loggedInIndex int,
```

Comment:
Are you using these changes somewhere else?  The change is fine, I'm just curious why it's in this PR.